sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -564,6 +564,9 @@ def test_pairwise_distances_reduction_is_usable_for():
     assert not BaseDistancesReductionDispatcher.is_usable_for(
         X_csr, Y_csr, metric="sqeuclidean"
     )
```

Comment:
Now that https://github.com/scikit-learn/scikit-learn/pull/24771 has been merged, this:
```suggestion
    assert BaseDistancesReductionDispatcher.is_usable_for(
        X_csr, Y_csr, metric="sqeuclidean"
    )
```

must pass.